packages-exp/messaging-exp/rollup.config.release.js

```diff
@@ -56,7 +55,10 @@ const es5Builds = [
   // sw builds
   {
     input: 'src/index.sw.ts',
-    output: [{ file: pkg.sw, format: 'es', sourcemap: true }],
+    output: [
+      { file: pkg.sw, format: 'cjs', sourcemap: true },
+      { file: pkg.sw, format: 'es', sourcemap: true }
+    ],
```

Comment:
I don't think you need es5 builds for sw since all browsers that support sw should support es2015+.